packages/renderer/MdToHtml.ts

```diff
@@ -378,12 +378,13 @@ export default class MdToHtml {
 	}
 
 	private async outputAssetsToExternalAssets_(output: any) {
-		for (const cssString of output.cssStrings) {
+		const newOutput = { ...output, pluginAssets: output.pluginAssets.slice() };
+		for (const cssString of newOutput.cssStrings) {
 			const filePath = await this.fsDriver().cacheCssToFile(cssString);
-			output.pluginAssets.push(filePath);
+			newOutput.pluginAssets.push(filePath);
 		}
-		delete output.cssStrings;
-		return output;
+		delete newOutput.cssStrings;
+		return newOutput;
```

Comment:
Thanks for updating this. Any change to add a test that would prove that modifying the output of this function doesn't modify the cached value?

Reply:
Yeah sure. There's a test file `packages/app-cli/tests/MdToHtml.ts`, should I make the necessary changes in that? Other than this there's no other test file related to `MdToHtml.ts`. I guess this is the test file that runs on running `yarn test` from root directory.

Reply:
Yes this file sounds good

Reply:
> Thanks for updating this. Any change to add a test that would prove that modifying the output of this function doesn't modify the cached value?

Done 👍